README.md

```diff
@@ -186,7 +186,7 @@ Kallax will generate one with the following signature:
 func NewUser(username string, password string, emails ...string) (*User, error)
 ```
 
-**IMPORTANT:** if your primary key is not auto-incrementable, you should set an ID for every model you create in your constructor. Or, at least, set it before inserting it.
+**IMPORTANT:** if your primary key is not auto-incrementable, you should set an ID for every model you create in your constructor. Or, at least, set it before saving it. Inserting, updating, deleting or reloading an object with no primary key set will return an error.
```

Comment:
Yes, not always, but that block is only referring to "not auto-incrementable" pks.  `if your primary key is not auto-incrementable`